scripts/softhsm_setup.sh

```diff
@@ -7,17 +20,17 @@ TOKEN_NAME="Demo Token"
 OBJECT_LABEL="Demo Object"
 PIN="0000"
 
-function install_dependencies() {
+install_dependencies() {
   # Install PKCS #11 related dependencies.
   # 1. softhsm2 is a software based HSM that implements the PKCS #11 spec.
   # 2. libp11-kit-dev contains a shared library at we will use to interact with
   # PKCS #11 device module, as well as pkcs11-tool which will be used for
   # interacting with the PKCS #11 module.
   # 3. gnutls-bin contains p11-tool which we will use to create PKCS #11 URIs.
-  sudo apt install softhsm2 libp11-kit-dev gnutls-bin
+  sudo apt install softhsm2 libp11-kit-dev gnutls-bin opensc-pkcs11
```

Comment:
```suggestion
  sudo apt install softhsm2 libp11-kit-dev gnutls-bin opensc
```

This is how I was able to figure out what the package name was:

1. `$ sudo docker run -it ubuntu:latest /bin/sh` create a docker container running a similar image as the GitHub Action. *Rest of commands are in docker container*
2. `$ apt update && apt install apt-file`  
3. `$ apt-file update`
4. `$ apt-file find pkcs11-tool` <---- This will find the correct package.

Reply:
Ahh okay, thank you!